src/examples/rest/src/backend/entities/mysql/api-key.ts

```diff
@@ -16,6 +17,6 @@ export class ApiKey extends BaseEntity implements ApiKeyStorage {
 	@Property({ type: Boolean, default: false })
 	revoked!: boolean;
 
-	@Property({ type: ArrayType, default: [] })
-	roles!: string[];
+	@Enum({ type: 'enumArray', items: () => Roles, array: true, default: [] })
```

Comment:
Do you need to include `array:true` here?

Reply:
Yes, if i remove `array: true`, i get error: `Expected Iterable`
<img width="1167" alt="Screenshot 2024-02-07 at 10 40 48 am" src="https://github.com/exogee-technology/graphweaver/assets/29393307/8615d867-9fde-4ed2-a6b5-1c817f770fa0">